app/helpers/profile_helpers.py

```diff
@@ -111,7 +111,7 @@ def _obtain_nb_points_per_segment_no_loss(distances, nb_points_total, total_dist
     nb_points_segments = []
     for d in distances:
         nb_points_segments.append(math.modf(max(nb_points_total * d / total_distance, 0.0)))
-    sum_int = sum([int(nbp[1]) for nbp in nb_points_segments])
+    sum_int = sum((int(nbp[1]) for nbp in nb_points_segments))
```

Comment:
Having had a quick look at: https://pylint.pycqa.org/en/latest/user_guide/messages/refactor/consider-using-generator.html, I guess, probably a
`sum_int = sum(int(nbp[1]) for nbp in nb_points_segments)` would do here, too.